personalInfoDashboard.js

```diff
@@ -0,0 +1,78 @@
+/*decleration */
+
+let firstName = "Mamikon";
+let lastName = "Gasparyan";
+let birthYear = 2005;
+let currentYear = 2025;
+let isStudent = true;
+let hobbies = ["Football", "MMA", "Cooking"];
+const contact = {
+    email: "mamikon.gasparyan.techx@gmail.com",
+    phone: "0xxxxxxxx",
+    city: "Yerevan",
+};
+
+/*basic console log*/
+
+console.log(`"Hi, my name is`,firstName, lastName,`. I live in `,contact.city,`"`);
+console.log(`"I am`,currentYear - birthYear,`years old and currently`,isStudent ? "a student" : "not a student",`"`);
+
+/*type coersion and comparison */
+
+let ageString = "25";
+let ageNumber = 25;
+
+console.log("Type coersion of == is", ageString == ageNumber);
+console.log("Type coersion of === is",ageString === ageNumber);
+console.log(`Type of ageNumber:`,typeof ageNumber,`, Type of ageString:`,typeof ageString);
+
+/*if else, switch,terinary operators and their functionality*/
+
+let score = 95;
+let grade = 'F';
+
+if(score >= 90 && score<=100){
```

Comment:
Always make sure to have proper formating
```suggestion
if (score >= 90 && score <= 100){
```